### PR TITLE
DRILL-7409: Check if BigInt is read correctly with parquet in diction…

### PR DESCRIPTION
**Pull request by denysord88**
…ary encoding.

### CODE REVIEW

**Comment by denysord88** [comment]
@Agirish could you please review the PR?

File: framework/resources/Functional/parquet_storage/dictionary/dictionary.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+      "verification-type": [
+        "in-memory"
+      ]
+    }
+  ],
+  "datasources": [
+    {
+      "mode": "cp",
+      "src": "Datasources/parquet_storage/bigIntDictionary.parquet",
+      "dest": "/drill/testdata/bigIntDictionary.parquet"
```
Comment (Agirish) [inline framework/resources/Functional/parquet_storage/dictionary/dictionary.json]: Can we please create a dir inside /drill/testdata? Like /drill/testdata/parquet/bigIntDictionary

Also, no need to provide /drill/testdata in the dest - any path specified is always relative to this location. For example, you can just provide parquet/bigIntDictionary

Comment (denysord88) [reply framework/resources/Functional/parquet_storage/dictionary/dictionary.json]: Done.

**Comment by denysord88** [comment]
@Agirish I moved test data, please review the changes.

**Pull request merged**